### DISCUSSION TITLE
Silent Times

### DISCUSSION THREAD

**Discussion by RestOp** [Ludy87/xplora_watch#158] [Ideas]
Hello,

i have found in API docs info about the Silent Times
![image](https://user-images.githubusercontent.com/46259141/217349285-8a17e0b0-2e7e-472b-90f4-7d60dca81397.png)

It it possible to use it in HA or is it till not not implementent in custom component?

Thanks a lot!

**Comment by Ludy87** [comment]
Silent is included in the integration, the problem what it gives is that it has the value of time but has no time zone dependency. 

It would have to be checked which time zone HA uses and which the clock uses, then correct the time with an offset if necessary.

I would have to plan some time for it to implement it if it is desired.

**Comment by RestOp** [comment]
Thanks!
I have got a sensor only after i have setuped the times, because oа it i have not found then before in HA.
